lib/chef-dk/policyfile/local_lock_fetcher.rb

```diff
@@ -111,12 +111,18 @@ def path
                 "The provided path #{source_options[:path]} does not exist.")
             end
           end
-          if !path.absolute?
-            path = Pathname.new(storage_config.relative_paths_root).join(path)
-          end
           path
         end
       end
+
+      def abs_path
+        source_path = Pathname.new(source_options[:path])
+        if !source_path.absolute?
```

Comment:
You should be able to just use `expand_path` here--it's a no-op if the path is already absolute:

```ruby
File.expand_path("/tmp/foo", "/home/bar")
# => "/tmp/foo"
```

Works the same for Pathnames

Reply:
updated